thirdparty/doctest/doctest.h

```diff
@@ -42,6 +42,10 @@
 #ifndef DOCTEST_LIBRARY_INCLUDED
 #define DOCTEST_LIBRARY_INCLUDED
 
+// CUSTOMIZATION TO PREVENT CRASHES IN MSVC COMPILER
+// YEAH I KNOW, IT TELLS ME ABOVE NOT TO EDIT IT, BUT I DID <3
+#include <sstream>
+
```

Comment:
Changes in the third-party code should be marked with comments:

```suggestion
// -- GODOT start --

// CUSTOMIZATION TO PREVENT CRASHES IN MSVC COMPILER
// YEAH I KNOW, IT TELLS ME ABOVE NOT TO EDIT IT, BUT I DID <3
#include <sstream>

// -- GODOT end --
```